test/foundry-tests/integration/RelayOrderReactorIntegration.t.sol

```diff
@@ -456,7 +456,55 @@ contract RelayOrderReactorIntegrationTest is GasSnapshot, Test, Interop, PermitS
         assertEq(tokenOut.balanceOf((feeRecipient)), 10 * USDC_ONE, "fee recipient balance");
     }
 
-    function test_execute_noActions_noInputs_noFee_succeeds() public {
+    /// @notice test an exact output V3 swap with a sweep of the tokenIn
+    function test_execute_exactOutputWithSweep() public {
+        ERC20 tokenIn = DAI;
+        ERC20 tokenOut = USDC;
+        // gas token diff than tokenIn to test the sweep
+        ERC20 gasToken = USDC;
```

Comment:
just a note but its definitely odd UX to have exact output swap where gasToken is in the same output token cause the end balance won't be the specified "exact"

Reply:
but nice test 👍